source/_posts/2020-03-14-using-embark-with-quorum.md

```diff
@@ -0,0 +1,387 @@
+5. [Run Embark and deploy a private contract](#5-run-embark-and-deploy-a-private-contract)
+6. [Sending a private transaction](#6-sending-a-private-transaction)
+7. [Extend web3.js](#7-extend-web3js)
+
+## 1. Initialise and run a Quorum network
+In order to get a Quorum network up quickly for development, the easiest path is to download the Quorum/Tessera binaries and use them to run the cloned 7nodes example.
+1. Download the latest Quorum binary from https://github.com/jpmorganchase/quorum/releases. Let's say it was downloaded to `/repos/jpmorganchase/quorum/build/bin/geth`.
+2. Update your `PATH` environment variable such that `geth` points to Quorum geth:
+```
+export PATH=/repos/jpmorganchase/quorum/build/bin:$PATH
+```
+3. Download the latest Tessera jar (`tessera-app-{version}-app.jar`, ie `tessera-app-0.10.3-app.jar`) from https://github.com/jpmorganchase/tessera/releases.
+4. Update your `TESSERA_JAR` environment variable to point to the downloaded Tessera jar:
+```
+export TESSERA_JAR=/path/to/tessera-app-0.10.3-app.jar
+```
+5. Clone the 7nodes example repo:
+```
+git clone https://github.com/jpmorganchase/quorum-examples
+cd examples/7nodes
+```
+6. Initialise Quorum using the desired consensus (`istanbul`, `raft`, `clique`):
+```
+# ./{consensus}-init.sh
+./istanbul-init.sh
+```
+7. Start Quorum using the desired consensus from step 6:
+```
+# ./{consensus}-start.sh tessera
+./istanbul-start.sh
+```
+  > **NOTE:** the default 7nodes example does not start Quorum geth with WebSockets. We can get around this by updating `./istanbul-start.sh` before executing it, as per https://github.com/jpmorganchase/quorum-examples/pull/221. As an alternative, clone [the fork with these changes](https://github.com/emizzle/quorum-examples) (on the `7nodes-websocket-support` branch) in Step 5 instead.
+
+After a few moments, you should have 7 Quorum nodes up and running.
+
+To stop the network:
+```
+./stop.sh
+```
+
+For more information on how to get the 7nodes example up and running, including information on using Vagrant and Docker, see https://github.com/jpmorganchase/quorum-examples.
+
+## 2. Install the `embark-quorum` plugin in your &ETH;App
+
+### Creating a &ETH;App if needed
+First, ensure you have a &ETH;App set up. We will be using the Embark's demo &ETH;App in this README, which can be created using `embark demo`:
+```
+# /bin/bash
+
+embark demo
+cd embark_demo
+```
+
+For more information on how to get started with the Embark Demo, view our [Quick Start guide](https://framework.embarklabs.io/docs/quick_start.html).
+
+### Installing `embark-quorum` in your &ETH;App
+To install `embark-quorum` in to our &ETH;App, we can add `embark-quorum` to our `packages.json`, then run `npm` or `yarn`:
+```
+# package.json
+"dependencies": {
+  "embark-quorum": "5.3.0"
+}
+```
+Then install the package with your package manager.
+```
+# /bin/bash
+npm i
+// OR
+yarn
+```
```

Comment:
This whole section can be replaced by: 
You can run:
```
yarn add embark-quorum
// OR
npm install embark-quorum --save
```
(add the @5.3.0) if you want people to be sure of the version.